src/loaders/common/loader_helper.py

```diff
@@ -70,12 +83,93 @@ def init_genome_atrri_doc(kbase_collection, load_version, genome_id):
     # The '_key' field for the document should be generated by applying a hash function to a combination of the
     # 'kbase_collection', 'load_version', and 'genome_id' fields.
     doc = {
-        names.FLD_ARANGO_KEY: md5_string(f"{kbase_collection}_{load_version}_{genome_id}"),
+        names.FLD_ARANGO_KEY: md5_string(
+            f"{kbase_collection}_{load_version}_{genome_id}"
+        ),
         names.FLD_COLLECTION_ID: kbase_collection,
         names.FLD_LOAD_VERSION: load_version,
         names.FLD_KBASE_ID: genome_id,
         # Begin with the input genome_id, though it may be altered by the calling script.
-        names.FLD_MATCHES_SELECTIONS: []  # for saving matches and selections
+        names.FLD_MATCHES_SELECTIONS: [],  # for saving matches and selections
     }
 
     return doc
+
+
+def get_token(token_filepath):
+    """
+    Get the token from the file and then from the env if it's not in the file.
+    """
+    if token_filepath:
+        with open(os.path.expanduser(token_filepath), "r") as f:
+            token = f.readline().strip()
+    else:
+        token = os.environ.get(KB_AUTH_TOKEN)
+        if not token:
+            raise ValueError(
+                f"Need to provide a token in the {KB_AUTH_TOKEN} "
+                + f"environment variable or as --token_filepath argument to the CLI"
+            )
+    return token
```

Comment:
```suggestion
    if token_filepath:
        with open(os.path.expanduser(token_filepath), "r") as f:
            token = f.readline().strip()
    else:
        token = os.environ.get(KB_AUTH_TOKEN)
    if not token:
        raise ValueError(
            f"Need to provide a token in the {KB_AUTH_TOKEN} "
            + f"environment variable or as --token_filepath argument to the CLI"
        )
    return token
```
This'll catch the case where the token file was empty as well

Reply:
Yeap, it will catch empty string. Updated!

Reply:
:+1: